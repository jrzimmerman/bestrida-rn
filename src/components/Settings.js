import React from 'react'
import {
  Text,
  View,
  StatusBar,
  StyleSheet
} from 'react-native'

const Settings = () => (
  <View style={styles.container}>
    <StatusBar barStyle="light-content" />
    <Text>Settings</Text>
  </View>
)

const styles = StyleSheet.create({
  container: {
    flex: 1,
    alignItems: 'center',
    paddingTop: 30
  }
})

export default Settings
