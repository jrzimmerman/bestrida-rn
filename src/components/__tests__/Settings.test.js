import 'react-native';
import React from 'react';
import { Settings } from '../Settings';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

test('renders Settings component', () => {
  const tree = renderer.create(
    <Settings
      dispatch={fn => fn}
    />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
