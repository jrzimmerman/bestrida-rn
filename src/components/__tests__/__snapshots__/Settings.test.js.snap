// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Settings component 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#2B2B2B",
      "flex": 1,
    }
  }
>
  <View
    accessibilityComponentType={undefined}
    accessibilityLabel={undefined}
    accessibilityTraits={undefined}
    accessible={true}
    collapsable={undefined}
    hitSlop={undefined}
    isTVSelectable={true}
    nativeID={undefined}
    onLayout={undefined}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignSelf": "stretch",
        "backgroundColor": "transparent",
        "borderColor": "#ef473a",
        "borderRadius": 4,
        "borderWidth": 1,
        "flexDirection": "row",
        "height": 40,
        "justifyContent": "space-around",
        "margin": 10,
        "marginTop": 20,
        "opacity": 1,
        "paddingHorizontal": 8,
        "paddingVertical": 6,
      }
    }
    testID={undefined}
    tvParallaxProperties={undefined}
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      disabled={false}
      ellipsizeMode="tail"
      style={
        Object {
          "alignSelf": "center",
          "color": "#ef473a",
          "flexDirection": "row",
          "fontSize": 14,
        }
      }
    >
      Logout
    </Text>
  </View>
</View>
`;
